DigitalLearningSolutions.Web/Scripts/trackingSystem/allDelegates.ts

```diff
@@ -0,0 +1,4 @@
+import { SearchSortAndPaginate } from '../searchSortAndPaginate/searchSortAndPaginate';
+
+// eslint-disable-next-line no-new
+new SearchSortAndPaginate('TrackingSystem/Delegates/All/AllDelegateItems');

```

Comment:
Note that the the name of this has updated to SearchSortFilterAndPaginate (and so has the folder path) in 533.